StepperEngine/src/Steps/CollectFilesInFolderStep.java

```diff
@@ -83,6 +83,19 @@ else if(input.getEffectiveName().equals(folderName.getEffectiveName())) {
         }
     }
 
+    @Override
+    public void setInputByName(DataType input, String inputName) {
+        if(inputName.equals(filter.getEffectiveName())) {
+            this.filter.setData((String) input.getData());
+            this.filter.setMandatory(false);
+        }
+        else if(inputName.equals(folderName.getEffectiveName())) {
+            this.folderName.setData((String) input.getData());
+            this.folderName.setMandatory(true);
```

Comment:
Why set the "isManadatory" again? Isn't it set in the constructor?

Reply:
you are right, i will change it on next pr